docs/source/api/nn/index.rst

```diff
@@ -14,3 +14,6 @@ Neural Networks
 	GraphNetworkBlock
 	GATConv
 	GCNConv
+	global_add_pool
```

Comment:
Idk why pooling stuff is here. Pooling is already merged to main so it shouldn't appear as new code 🤔 any idea @francescofarina ?